torch/csrc/jit/passes/tensorexpr_fuser.h

```diff
@@ -69,5 +69,9 @@ TORCH_API bool isSupported(Node* node);
 /// @return Reference of the custome operator set
 ///
 TORCH_API OperatorSet& getCustomOperatorSet();
+
 } // namespace tensorexpr
 } // namespace torch::jit
+
+C10_DECLARE_bool(torch_jit_disable_cat);
```

Comment:
What's the reason for this?

Reply:
Because `C10_DEFINE_bool(torch_jit_disable_cat,...` is used inside torch/csrc/jit/passes/tensorexpr_fuser.cpp. We want to add the corresponding declaration to avoid complier warnings.